### PR TITLE
Canary status latency graphs

### PR DESCRIPTION
**Pull request by xavi-**
This PR is probably easy to review going commit by commit.

- Fixed lint errors
- Now ECS metrics page falls back to cache if ajax request to ecs-metrics fails
- Cache is updated once a minute every minute
- Added latency and status-code graphs for canaries

### CODE REVIEW

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -1884,7 +1899,222 @@ function createProcessMetricsPanels(env, app) {
...
     "titleSize": "h6"
   };
-};
+}
+
+function createESStatusCodes(env, app, isCanary) {
+  if(isCanary) { app += "-canary"; }
+
+  return {
+    "title": "Status Codes (Elasticsearch)",
```
Comment (bgveenstra) [inline dashboards/scripted_services.js]: The significance of the "(Elasticsearch)" part of this title wasn't immediately clear to me. 

Are you using Elasticsearch in the titles to distinguish from the ALB status and latency panels that are Cloudwatch-backed?

There might be a way to rephrase the title to make it clearer what the fact the data is coming from Elasticsearch means for the engineer looking at the data, but I can't think of one right now!

Comment (xavi-) [reply dashboards/scripted_services.js]: Yep, that's correct reading.  I also agree that it's not the best phrasing.  Of course, I'm open to suggestions.  Maybe "Latency via Elasticsearch"? or "Latency according to Elasticsearch"?

**Comment by xavi-** [comment]
Yep yep, your understanding is correct, and it highlights and issue I overlooked.  I need to add a switch to use "Elasticsearch Dev" for non-prod apps.

File: dashboards/scripted_services.js (hunk 1 above)
Comment (xavi-) [reply dashboards/scripted_services.js]: After thinking about it, the fact the data comes from Elasticsearch feels like an implementation detail that doesn't need to be communicated.  If people get confused we can always add it back.

**Pull request merged**